openapiGenerator.go

```diff
@@ -455,7 +446,9 @@ func (g *openapiGenerator) generateMessageSchema(message *protomodel.MessageDesc
 			tmp := getSoloSchemaForMarkerType(schemaType)
 			schema := getSchemaIfRepeated(&tmp, repeated)
 			schema.Description = fieldDesc
-			g.mustApplyRulesToSchema(fieldRules, schema, markers.TargetField)
+			// Filter out Type markers since we've already applied the type
+			filteredRules := g.filterOutTypeMarkers(fieldRules)
+			g.mustApplyRulesToSchema(filteredRules, schema, markers.TargetField)
```

Comment:
why is this change required?

Reply:
sorry this isn't I removed this change - the type value `ApplyToSchema` function was incorrect.